pkg/aggregator/no_orchestrator.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !orchestrator
+
+package aggregator
+
+// Orchestrator Explorer is enabled by default but
+// the forwarder is only created if the orchestrator
+// build tag exists
+
+// OrchestratorForwarderSupport shows if the orchestrator build tag is enabled
+const OrchestratorForwarderSupport = false
```

Comment:
Would it make sense to make this const private?

```suggestion
const orchestratorForwarderSupport = false
```

Reply:
[e08d6ce](https://github.com/DataDog/datadog-agent/pull/25027/commits/e08d6cea6c381a68caa8bb7ba0ea8b47e1349dce)